checker_test.go

```diff
@@ -970,6 +975,179 @@ got args:
 want args:
   regexp
 `,
+}, {
+	about:   "ErrorAs: exact match",
+	checker: qt.ErrorAs,
+	got:     error(errBadWolf),
```

Comment:
I guess just errBadWolf is sufficient? It will be received as an empty interface anyway. Same in the occurrences below.

Reply:
You're correct.

I ran into an issue where I needed to explicitly coerce `errBadWolf` to an `error`, but that code got removed and apparently I continued propagating it. Fixed!